lib/easypost/util.rb

```diff
@@ -293,4 +293,57 @@ def self.get_lowest_stateless_rate(stateless_rates, carriers = [], services = []
   def self.receive_event(raw_input)
     EasyPost::InternalUtilities::Json.convert_json_to_object(JSON.parse(raw_input), EasyPost::Models::EasyPostObject)
   end
+
+  def self.get_lowest_smartrate(smartrates, delivery_days, delivery_accuracy)
```

Comment:
Per the our convo on Slack, this is something we needed to watch for was correcting usages of `smartrate` to `smart_rate` since it's two words. True of this whole code block and library

```suggestion
  def self.get_lowest_smar_trate(smart_rates, delivery_days, delivery_accuracy)
```

Reply:
also, `smart_rate`, not `smar_trate`